errorprone-checks/src/main/java/org/hyperledger/errorpronechecks/BannedMethod.java

```diff
@@ -44,7 +44,9 @@ public class BannedMethod extends BugChecker implements MethodInvocationTreeMatc
           staticMethod().onClass("com.google.common.base.Objects").withAnyName(),
           "Do not use com.google.common.base.Objects methods, use java.util.Objects methods instead.",
           staticMethod().onClass("org.junit.Assert"),
-          "Do not use junit assertions. Use assertj assertions instead.");
+          "Do not use junit assertions. Use assertj assertions instead.",
+          staticMethod().onClass("org.apache.logging.log4j.LogManager"),
+          "Do not use org.apache.logging.log4j.LogManager, use org.slf4j.LoggerFactory methods instead.");
```

Comment:
Should this include the other log4j methods that were removed in this PR? static methods on `org.apache.logging.log4j.core.config.Configurator`, `org.apache.logging.log4j.ThreadContext`, instance methods on import `org.apache.logging.log4j.core.LoggerContext`, `org.apache.logging.log4j.core.config.Configuration`,
`org.apache.logging.log4j.core.config.LoggerConfig`?  Or is there a way to go full Ender-Wiggin on it and have it fail for any class from the log4j packages?

Reply:
No, we cannot rip off the whole package because we still have hard dependencies on it. The most visible one is `o.h.b.util.Log4j2ConfiguratorUtil` but we also depend on `o.a.l.log4j.Level` for the CLI. I went on banning just `o.a.l.log4j.LogManager` b/c it's the most common entry point to the Log4j framework from the day to day developer perspective